packages/bind.rb

```diff
@@ -3,23 +3,23 @@
 class Bind < Package
   description 'BIND is open source software that enables you to publish your Domain Name System (DNS) information on the Internet, and to resolve DNS queries for your users.'
   homepage 'https://www.isc.org/downloads/bind/'
-  version '9.19.9'
+  version '9.18.16'
   license 'Apache-2.0, BSD, BSD-2, GPL-2, HPND, ISC and MPL-2.0'
-  compatibility 'all'
+  compatibility 'x86_64'
```

Comment:
Why was this changed?

Reply:
Oops. That's a bug from me not fixing up a file...

Reply:
Fixed.